symfony/plugins/orangehrmTimePlugin/service/CustomerService.php

```diff
@@ -98,80 +128,114 @@ public function getActiveCustomerById($customerId) {
      * @param type $customerName
      * @return type
      */
-    public function getCustomerByName($customerName) {
+    public function getCustomerByName($customerName)
```

Comment:
some of existing methods we may have to remove here and unit testing

Reply:
done